scripts/default_config.py

```diff
@@ -616,7 +616,7 @@ def model_kwargs(cfg, num_classes):
         'type': cfg.model.type,
         'self_challenging_cfg': cfg.model.self_challenging_cfg,
         'hidden_dim': cfg.model.transformer.hidden_dim,
-        'similarity_adjustment': cfg.loss.am_binary.amb_t > 0,
+        'similarity_adjustment': cfg.loss.am_binary.amb_t != 1.,
```

Comment:
Why do you change the condition? amb_t may vary from 1 to 7

Reply:
If amb_t is 1, then we shouldn't perform sym_adjust() call because this function is identity, in other cases we can call it.

Reply:
It is also useless with t = 0 since the value would be always 1. , I propose to change it to `amb_t > 1.`